lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -80,8 +128,37 @@ public static double addNumbers(double[] values) {
    * @param n The first `n` of fibonacci values to compute.
    * @return An array containing the first `n` fibonacci values.
    */
+
+  // Validate Input: Ensure n is positive. Return an empty array for invalid input.
+
+  // Initialize Array: Create an array of size n to store Fibonacci numbers.
+
+  // Handle Base Cases: Directly assign 1 to the first (and second if n > 1) elements.
+
+  // Calculate Fibonacci: Loop from the third element, each value is the sum of the two preceding
+  // ones.
+
+  // Return Array: Output the array with the first n Fibonacci numbers.
   public static int[] getFirstNFibonacciNumbers(int n) {
-    return null;
+
+    if (n < 1) {
```

Comment:
Try doing all of this in just a single `for` loop.

Reply:
hmm interesting will do !